pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -273,8 +273,12 @@
   hardware_prizes_form_school_type_option_middle_school: 'Middle school'
   hardware_prizes_form_school_type_option_high_school: 'High school'
   hardware_prizes_form_qualifying_school_label: 'This is a K-12 US public school'
+  hardware_prizes_form_title_I_school_label: This school qualifies for Title I funding
```

Comment:
Not really, I usually don't add them unless they need them. Would it be better to be consistent and add them?